packages/map-template/src/components/Directions/Directions.jsx

```diff
@@ -127,6 +148,12 @@ function Directions({ isOpen, onBack, onSetSize, snapPointSwiped, onRouteFinishe
                 } else {
                     setDestinationDisplayRule(mapsIndoorsInstance.getDisplayRule(directions.destinationLocation));
                 }
+
+                if (mapType === 'mapbox') {
```

Comment:
These lines look very similar to the content of the setMinZoom function. Can we use that?